docs/ota.md

```diff
@@ -63,7 +73,7 @@ Notify PTA BO that the bus is starting a block.
 | Name             | Type    | Description                   |
 |------------------|---------|-------------------------------|
 | eventTimestamp   | string  | ISO 8601, UTC                 |
-| vehicleNumber    | integer | same as for the bridged topic |
+| vehicleNumber    | integer | Same as the &lt;vehicleid&gt; in the topic, a VIN |
```

Comment:
I don't think this is correct any more. It is not an integer, but a string. 

In general we should request @andrelin or someone else from sanntid to review the documentation for sign-on and sign-off, as they have better insights into this format than me.

Reply:
Version 2.1 pdf has string as the type so it’s just an omission that should be corrected.

Reply:
@phototropic is correct that this was an oversight by me. It has been corrected.